client/__mocks__/@react-native-firebase/firestore.js

```diff
@@ -0,0 +1,14 @@
+const mockFirestore = {
```

Comment:
Maybe possible and better to use the real emulator with "seeded mock data" instead 🤓